proto/pbpeering/peering.go

```diff
@@ -14,6 +15,7 @@ import (
 	"github.com/hashicorp/consul/agent/structs"
 	"github.com/hashicorp/consul/api"
 	"github.com/hashicorp/consul/lib"
+	"github.com/hashicorp/consul/lib/stringslice"
```

Comment:
Stray import?

Reply:
Yep! From my previous attempt at hand-rolling it, will remove.